tests/zfs-tests/tests/functional/rsend/rsend_019_pos.ksh

```diff
@@ -35,6 +35,11 @@
 
 verify_runnable "both"
 
+# See issue: https://github.com/zfsonlinux/zfs/issues/6086
+if is_32bit; then
```

Comment:
Is this change in the wrong file? According to your comments, you intended to skip `rsend_020_pos`.